developer-hub/src/polkadot_sdk/mod.rs

```diff
@@ -0,0 +1,138 @@
+//! [![Runtime](https://img.shields.io/badge/fellowship-runtimes-e6007a?logo=polkadot)](https://github.com/polkadot-fellows/runtimes)
+//! [![Manifesto](https://img.shields.io/badge/fellowship-manifesto-e6007a?logo=polkadot)](https://github.com/polkadot-fellows/manifesto)
+//!
+//! ## Getting Started
+//!
+//! The primary way to get started with the Polkadot SDK is to start writing a Substrate-based
+//! runtime using FRAME. See:
+//!
+//! * [`polkadot`], to understand what is Polkadot as a development platform.
+//! * [`substrate`], for an overview of what Substrate is.
+//! * Jump right into [`frame`] to learn about how to write a FRAME-based blockchain runtime.
+//! * Continue with the [`developer_hub`'s "getting started"](crate#getting-started).
+//!
+//! ## Structure
+//!
+//! #### Substrate
+//!
+//! [![Substrate-license](https://img.shields.io/badge/License-GPL3%2FApache2.0-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/LICENSE-APACHE2)
+//! [![GitHub Repo](https://img.shields.io/badge/github-substrate-2324CC85)](https://github.com/paritytech/polkadot-sdk/blob/master/substrate)
+//!
+//! [`substrate`] is the base blockchain framework used to power the Polkadot SDK. It is a full
+//! toolkit to create sovereign blockchains, including but not limited to those who connect to
+//! Polkadot as parachains.
+//!
+//! #### FRAME
+//!
+//! [![Substrate-license](https://img.shields.io/badge/License-Apache2.0-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/LICENSE-APACHE2)
+//! [![GitHub Repo](https://img.shields.io/badge/github-frame-2324CC85)](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame)
+//!
+//! [`frame`] is the framework used to create Substrate-based runtimes. Learn more about the
+//! distinction of a runtime and client in [`crate::reference_docs::wasm_meta_protocol`].
+//!
+//! #### Cumulus
+//!
+//! [![Cumulus-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-cumulus-white)](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus)
+//!
+//! [`cumulus`] transforms FRAME-based runtimes into Polkadot-compatible parachain runtimes, and
+//! Substrate-based clients into Polkadot-compatible collators.
+//!
+//! #### Polkadot
+//!
+//! [![Polkadot-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-polkadot-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot)
+//!
+//! [`polkadot`] is an implementation of a Polkadot client in Rust, by `@paritytech`. The Polkadot
+//! runtimes are located under the [`polkadot-fellows/runtimes`](https://github.com/polkadot-fellows/runtimes) repository.
```

Comment:
Maybe not in scope, but we can link to the fellowship via the Wiki: https://wiki.polkadot.network/docs/learn-polkadot-opengov#the-polkadot-technical-fellowship 

or a glossary definition: https://wiki.polkadot.network/docs/glossary#fellowship

Reply:
sure, feel free to add it.